src/modules/todo/api/index.ts

```diff
@@ -0,0 +1,14 @@
+import { TodoItem } from '../components/todo/Todo.interface'
+
+export const fetchTodos = () => {
+  return fetch('api/todos').then(async (res) => {
+    const { todos } = (await res.json()) as { todos: TodoItem[] }
+    return todos
+  })
+}
+
+type PostTodoData = Omit<TodoItem, 'id'>
+
+export const postTodo = (data: PostTodoData) => {
```

Comment:
I think it would be cood idea to create const with next methods:
`const TodoApiService = {
retrieve: ....
create: ....
}
`

the reason for this, that In that case we unify our Api request, so we would have `retrieve`, `list`, `update`, `create` and `delete` As this is most commen methods that we could have.